addons/website_blog/tests/test_ui.py

```diff
@@ -52,3 +52,6 @@ def test_autocomplete_with_date(self):
         self.env.ref('website_blog.opt_blog_sidebar_show').active = True
         self.env.ref('website_blog.opt_sidebar_blog_index_follow_us').active = False
         self.start_tour("/blog", 'blog_autocomplete_with_date')
+
+    def test_avatar_comment(self):
+        self.start_tour(self.env['website'].get_client_action_url('/blog'), 'blog_avatar_comment', login='admin')
```

Comment:
Personnally I don't think we have to add a tour just for that: tour takes a lot of time to launch and run, runbot is already taking ages to build. Can't you just add a unit test accessing the image route directly ?